engine/core/src/main/java/org/datacleaner/util/SchemaComparator.java

```diff
@@ -53,7 +54,7 @@ public static boolean isInformationSchema(Schema schema) {
 		if (name == null) {
 			return false;
 		}
-		return "information_schema".equals(name.toLowerCase());
+		return MetaModelHelper.isInformationSchema(schema);
```

Comment:
MetaModelHelper already does the name check and lowercasing, so no need to repeat it.